muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/evaluation/body/application/evaluation/editors/supplementation-editor.tsx

```diff
@@ -270,17 +269,21 @@ class SupplementationEditor extends SessionStateComponent<
          * onSuccess
          */
         onSuccess: () => {
-          cleanWorkspaceAndSupplementationDrafts(this.state.draftId);
+          /* cleanWorkspaceAndSupplementationDrafts(this.state.draftId); */
```

Comment:
Remove unused commented code blocks or mark them if they are meant to be used in the future.